runtime-infra/pn-ipc.yaml

```diff
@@ -260,6 +269,71 @@ Resources:
                   - !GetAtt SafeStorageToDeliveryPushQueue.Outputs.QueueARN
                   - !GetAtt "SafeStorageToDowntimeLogsQueue.Outputs.QueueARN"
   
+  # Resource based policy to allow EventBridge to publish to CloudWatch Logs
+  LogGroupCoreEventBusPolicy:
+    Type: AWS::Logs::ResourcePolicy
+    Properties:
+      PolicyName: !Sub "${ProjectName}-CoreEventBusToCWLogsPolicy"
+      PolicyDocument: !Sub |
```

Comment:
Avere questo policy document in yaml sarebbe più carino

Reply:
@marco-vit-pagopa il tipo PolicyDocument è di tipo string, non ci sono alternative per definire una risorsa `AWS::Logs::ResourcePolicy`

Reply:
Temevo questa risposta :(